subsys/bluetooth/mesh/access.c

```diff
@@ -1695,7 +1695,7 @@ int bt_mesh_model_extend(struct bt_mesh_model *extending_mod, struct bt_mesh_mod
 		a->next = b;
 	}
 
-
+end:
```

Comment:
I'd rename this to `register:` - makes the code flow a bit more readable than a generic "goto end"

Reply:
Changed to 'register_extension'!